debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -360,6 +366,38 @@ protected Object convertDecimal(Column column, Field fieldDefn, Object data, Dec
         return SpecialValueDecimal.fromLogical(new SpecialValueDecimal(newDecimal), mode, column.name());
     }
 
+
+    protected Object convertHStore(Column column,Field fieldDefn,Object data,HStoreMode mode){
+        if(mode == HStoreMode.MAP){
+            return convertToMap(column,fieldDefn,data);
+        }
+        else if(mode == HStoreMode.JSON){
+            return super.convertString(column,fieldDefn,data);
+        }
+           return super.convertString(column,fieldDefn,data);
+    }
+
+    private Object convertToMap(Column column,Field fieldDefn,Object data){
+        if(data == null){
+            data = fieldDefn.schema().defaultValue();
+        }
+        if(data == null){
+            return new Map<String,String>;
```

Comment:
Looks like this won't compile without `()` :)

Reply:
Fixed it .